app/controllers/warehouse_reports/cas/chronic_reconciliation_controller.rb

```diff
@@ -25,7 +26,7 @@ def index
       @not_on_list = client_source.
         where(id: (cas_ids - chronic_ids)).
         order(last_name: :asc, first_name: :asc).
-        includes(:chronics)
+        includes(site_chronics_sym)
```

Comment:
Any of these calls to `site_chronics_sym` that are in active record queries will probably be more readable with `site_chronics_table`